spec/controllers/admin/organizations/invoices_controller_spec.rb

```diff
@@ -70,21 +68,32 @@
       end
       context "with amount_do 0" do
         it "makes the invoice active" do
-          expect do
-            post :create, organization_id: organization.to_param, invoice: params.merge(amount_due: "0", end_at: "2020-09-05T23:00:00", child_paid_feature_slugs: %[abandoned_bike_messages passwordless_users])
-          end.to change(Invoice, :count).by 1
-          # TODO: Rails 5 update
-          organization.update_attributes(updated_at: Time.current)
-          invoice = organization.invoices.last
-          expect(invoice.active?).to be_truthy
-          expect(organization.is_paid).to be_truthy
-          expect(organization.paid_feature_slugs).to eq(%w[abandoned_bike_messages passwordless_users])
-          expect(invoice.paid_feature_ids).to match_array([paid_feature1.id, paid_feature2.id])
-          expect(invoice.amount_due).to eq 0
-          # TimeParser isn't storing records perfectly - for now, just ignoring since fix can be separate
-          expect(invoice.subscription_start_at.to_i).to be_within(1.day).of 1536202800
-          expect(invoice.subscription_end_at.to_i).to be_within(1.day).of 1536202800 + 2.years
-          expect(invoice.notes).to eq params[:notes]
+          Sidekiq::Testing.inline! do
```

Comment:
Execute this test synchronously in order to test after_commit callbacks without manually triggering them.